shared/AppInsightsCore/src/JavaScriptSDK/BaseTelemetryPlugin.ts

```diff
@@ -3,16 +3,18 @@
 
 "use strict";
 
+import dynamicProto from "@microsoft/dynamicproto-js";
 import { IAppInsightsCore } from "../JavaScriptSDK.Interfaces/IAppInsightsCore"
 import { IConfiguration } from "../JavaScriptSDK.Interfaces/IConfiguration";
 import { IDiagnosticLogger } from "../JavaScriptSDK.Interfaces/IDiagnosticLogger";
 import { IPlugin, ITelemetryPlugin } from "../JavaScriptSDK.Interfaces/ITelemetryPlugin";
 import { ITelemetryItem } from "../JavaScriptSDK.Interfaces/ITelemetryItem";
 import { IProcessTelemetryContext } from "../JavaScriptSDK.Interfaces/IProcessTelemetryContext";
 import { ITelemetryPluginChain } from "../JavaScriptSDK.Interfaces/ITelemetryPluginChain";
-import { ProcessTelemetryContext } from "./ProcessTelemetryContext";
-import { isFunction, isNullOrUndefined, setValue } from "./HelperFuncs";
+import { createProcessTelemetryContext } from "./ProcessTelemetryContext";
```

Comment:
While the `ProcessTelemetryContext` class still exists (for backward compatibility only), changing to no longer directly use it for minification. The class itself uses the `createProcessTelemetryContext` to generate it's actual usage